tests/testapp1/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class Testapp1Config(AppConfig):
     name = "tests.testapp1"
+
+    def ready(self):
+        from . import tasks
```

Comment:
I think you'll have to add a `# noqa` to the end of this line and to the other import in `models.py` to appease the linter.